Sources/Node/SchemaWrapper/SchemaWrapper.swift

```diff
@@ -0,0 +1,17 @@
+public protocol SchemaWrapper:
+    NodeConvertible,
+    PathIndexable,
+    Polymorphic,
+    Equatable,
+    ExpressibleByNilLiteral,
+    ExpressibleByBooleanLiteral,
+    ExpressibleByIntegerLiteral,
+    ExpressibleByFloatLiteral,
+    ExpressibleByStringLiteral,
+    ExpressibleByArrayLiteral,
+    ExpressibleByDictionaryLiteral
+{
+    var schema: Schema { get set }
```

Comment:
I see what you mean.

```swift
makeValue() -> Value
```

That's kind of confusing.

If we could prefix the `Node` module name, that would be nice. But we can't since there is also a `Node` type.